background/main.ts

```diff
@@ -631,6 +647,16 @@ export default class Main extends BaseService<never> {
     )
   }
 
+  async connectSigningService(): Promise<void> {
+    this.keyringService.emitter.on("address", (address) =>
+      this.signingService.addTrackedAddress(address, "keyring")
+    )
+
+    this.ledgerService.emitter.on("address", (obj) =>
+      this.signingService.addTrackedAddress(obj.address, "ledger")
+    )
```

Comment:
By using desctructuring you can avoid the `obj` variable

```suggestion
    this.ledgerService.emitter.on("address", ({ address }) =>
      this.signingService.addTrackedAddress(address, "ledger")
    )
```

Otherwise, choose either: a more specific name (e.g. `addressData`, `account`, etc., depending of the name of the type) or a better generic name (e.g. `event`).

Reply:
Fixed, thanks!